WordPress/Classes/ViewRelated/Post/Controllers/AbstractPostListViewController.swift

```diff
@@ -852,24 +856,7 @@ class AbstractPostListViewController: UIViewController,
     // MARK: - Misc
 
     private func showRefreshingIndicator() {
-        guard navigationItem.titleView == nil else {
-            return
-        }
-
-        let spinner = UIActivityIndicatorView(style: .medium)
-        spinner.startAnimating()
-        spinner.tintColor = .secondaryLabel
-        spinner.transform = .init(scaleX: 0.8, y: 0.8)
-
-        let titleView = UILabel()
-        titleView.text = Strings.updating + "..."
-        titleView.font = UIFont.preferredFont(forTextStyle: .headline)
-        titleView.textColor = UIColor.secondaryLabel
-
-        let stack = UIStackView(arrangedSubviews: [spinner, titleView])
-        stack.spacing = 8
-
-        navigationItem.titleView = stack
+        // TODO: Design a better way to show it that doesn't affect the navigation bar
```

Comment:
I think we need to design a good way to indicate that the locally cached content is refreshing. The previous solution was no longer working on iOS 26 as everything begins jumping around.

Reply:
Should this code be kept for non-iOS 26?